### DISCUSSION TITLE
Another Gamemode inside identity mod

### DISCUSSION THREAD

**Discussion by kazzybun** [Draylar/identity#151] [Ideas]
Could u make it that there's gamemode were u are like in spectator but u cant go thru walls basiclly like ghost
and u can possess other mobs instand needing too kill them too transform? and u cant like pop out of their body and switch too another one or like make it that ur normal player but u can pop out of ur body and become ghost for as long u want and possess other mobs and when ur done u need too goo back too ur body or press botton too back too ur body and u cant goo too far in ghost form

**Comment by Shadowysn** [comment]
Another mod does most of what you want, called the Requiem mod.
By default you can only possess mobs with no soul (undead mobs), but from what I heard it is configurable to allow you to possess any mob.
https://www.curseforge.com/minecraft/mc-mods/requiem
